cluster-quorum-xml/src/main/java/ClusterQuorum.java

```diff
@@ -4,6 +4,34 @@
 import com.hazelcast.core.IQueue;
 import com.hazelcast.quorum.QuorumException;
 
+/**
+ * The term "quorum" simply refers to the count of members in the cluster required for an operation to succeed.
+ * It does NOT refer to an implementation of Paxos or Raft protocols as used in many NoSQL and distribuetd systems.
+ * The mechanism it provides in Hazelcast protects the user in case the number of nodes in a cluster drops below the specified one.
+ *
+ * Hazelcast Quorum is supported in the following data-structures:
+ * - Map
```

Comment:
Should be `IMap`

Reply:
Also `ICache`, `IQueue` and `ILock`.

Reply:
Thx, I also fixed some others, like RingBuffer (not IRingbuffer, etc.)

Reply:
Good catch, but actually it's `Ringbuffer` not `RingBuffer` :D